tests/c/csound_debugger_test.cpp

```diff
@@ -174,7 +180,10 @@ TEST_F (DebuggerTests, testBreakpointInstrument)
 int count = 0;
 static void brkpt_cb5(CSOUND *csound, debug_bkpt_info_t *bkpt_info, void *userdata)
 {
-    debug_opcode_t *debug_opcode = bkpt_info->currentOpcode;
+    (void)(csound);
+    (void)(bkpt_info);
+    (void)(userdata);
+    // debug_opcode_t *debug_opcode = bkpt_info->currentOpcode;
```

Comment:
Whats the warning here?

Reply:
debug_opcode was unused. Maybe it's needed for breakpoint or something though?